src/convertToObject.js

```diff
@@ -3,10 +3,32 @@
 /**
  * @param {string} sourceString
  *
- * @return {object}
+ * @return {object}n
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+function convertToObject(stylesString) {
+  // Создаем копию параметра, чтобы не изменять его
+  const modifiedStylesString = stylesString
+    .replace(/;+\s*$/, '') // Удаляет точки с запятой в конце строки
```

Comment:
The use of `replace` here to remove trailing semicolons is not necessary. You can handle empty rules after splitting the string into individual declarations. This would simplify the logic and make the code cleaner.